util/libcrypto.num

```diff
@@ -5163,14 +5159,16 @@ OSSL_DESERIALIZER_settable_ctx_params   ?	3_0_0	EXIST::FUNCTION:
 OSSL_DESERIALIZER_CTX_new               ?	3_0_0	EXIST::FUNCTION:
 OSSL_DESERIALIZER_CTX_set_params        ?	3_0_0	EXIST::FUNCTION:
 OSSL_DESERIALIZER_CTX_free              ?	3_0_0	EXIST::FUNCTION:
+OSSL_DESERIALIZER_CTX_set_cipher        ?	3_0_0	NOEXIST::FUNCTION:
 OSSL_DESERIALIZER_CTX_set_passphrase    ?	3_0_0	EXIST::FUNCTION:
-OSSL_DESERIALIZER_CTX_set_pem_password_cb ?	3_0_0	EXIST::FUNCTION:
+OSSL_DESERIALIZER_CTX_set_passphrase_cb ?	3_0_0	NOEXIST::FUNCTION:
```

Comment:
Not sure this file is supposed to look like this..

Reply:
No idea where this came from 😭  I've reverted the commit in question and rerun `make update`.

Reply:
No change.  I'll revert and just do the errors.

Reply:
Rebased to master, looks like libcrypto.num skipped an update.